README.md

```diff
@@ -185,12 +268,42 @@ and [development](https://docs.ansible.com/ansible/latest/dev_guide/index.html).
 * File headers and functions should have comments for their intent.
 
 ## Ansible Best Practices
-
+* Ansible variables use lazy evaluation. [more_info](https://github.com/ansible/ansible/issues/10374)
+* All tags should be namespaced/prefixed with the role name. 
+* Use the `command` module instead of the `shell` module when use of either is required.
+  In all other cases use a dedicated module, if it exists. If not, see the [section](#check-mode-and-idempotency-issues) 
```

Comment:
I think you did not address @greg-hellings concerns here. Can you put back the original wording, since it seems it is clearer, please?